# Deskripsi Data dan File

## Dataset yang digunakan
Nama dataset : vgsales.csv<br>format dataset : CSV<br>Dimensi data : 16598 baris x 11 kolom<br>Ukuran file data : 1.35 MB<br>
Deskripsi:<br>Dataset yang digunakan adalah mengenai penjualan video games yang berisi daftar video game dengan penjualan lebih dari 100.000 eksemplar. Data set ini diambil dari https://www.kaggle.com/datasets/gregorut/videogamesales/data

## Informasi yang ingin diketahui
1. Range penjualan video games secara global.
2. Range penjualan video games di setiap daerah.
3. 10 game pertama dengan penjualan global paling banyak.
4. 10 data pertama game dengan penjualan global paling sedikit.
5. Distribusi frekuensi platform game.
6. Distribusi frekuensi genre game.
7. Jumlah salinan game yang terjual untuk setiap genrenya pada setiap region.
8. 10 studio game dengan penjualan game paling sedikit.
9. 10 studio game  dengan penjualan game terbanyak.
10. Visualisasi data perbandingan jumlah Global Sales untuk sepuluh Publisher teratas.
11. Visualisasi data penjualan suatu platform selama waktu tertentu.
12. Visualisasi data pembagian penjualan game dari perusahaan platform tertentu.
13. Visualisasi data mengenai relasi antara penjualan di kedua negara tertentu.
14. Visualisasi data mengenai relasi antara penjualan global game dengan tahun penjualan game.
15. Nilai korelasi antara jumlah penjualan di masing-masing negara.

## Import dan membaca data

In [3]:
import pandas as pd
import matplotlib.pyplot as plt

df = pd.read_csv("vgsales.csv")

## Mencari dimensi data

Ada 2 cara untuk mencari dimensi data, cara pertama adalah dengan cara print dataset secara keseluruhan lalu pada bagian paling akan tercantum berapa dimensinya. Sedangkan, cara kedua adalah dengan cara membuat program menghitung panjang baris dan kolom pada data.

In [4]:
# Cara 1:
print(df)

        Rank                                              Name Platform  \
0          1                                        Wii Sports      Wii   
1          2                                 Super Mario Bros.      NES   
2          3                                    Mario Kart Wii      Wii   
3          4                                 Wii Sports Resort      Wii   
4          5                          Pokemon Red/Pokemon Blue       GB   
...      ...                                               ...      ...   
16593  16596                Woody Woodpecker in Crazy Castle 5      GBA   
16594  16597                     Men in Black II: Alien Escape       GC   
16595  16598  SCORE International Baja 1000: The Official Game      PS2   
16596  16599                                        Know How 2       DS   
16597  16600                                  Spirits & Spells      GBA   

         Year         Genre   Publisher  NA_Sales  EU_Sales  JP_Sales  \
0      2006.0        Sport

Dari output diatas, dapat kita ketahui bahwa dimensi dari dataset ini adalah 16598 baris x 11 kolom

In [9]:
# Cara 2:
panjang_baris = len(df)
panjang_kolom = len(df.columns)
print("Dimensi data:", str(panjang_baris), "baris x", str(panjang_kolom), "kolom")

Dimensi data: 16598 baris x 11 kolom


## Mencari ukuran file data
Selain dengan cara melihat pada properties data itu sendiri, adapun cara lain untuk mencari ukuran file data dengan cara menggunakan program

In [10]:
import os
statinfo = os.stat("vgsales.csv")
print("Ukuran file data tersebut sebesar", statinfo.st_size, "byte")

Ukuran file data tersebut sebesar 1355781 byte


Sehingga dapat kita dapatkan besar ukuran file data tersebut sebesar 1355781 byte (1.35MB)